letsencrypt-win-simple/Plugin/SanByDomainPlugin.cs

```diff
@@ -0,0 +1,288 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using Serilog;
+using Microsoft.Web.Administration;
+
+namespace LetsEncrypt.ACME.Simple
+{
+    public class SanByDomainPlugin : Plugin
+    {
+        public override string Name => "SanByDomain";
+        private const int SanMax = 100;
```

Comment:
I like this but I think it should be moved to the main Program and can then be used by the other plugins.